build.sh

```diff
@@ -29,6 +29,8 @@ PRINT_USAGE() {
     echo "  -j [N], --jobs[=N]   Multicore build, allow N jobs at once"
```

Comment:
Oh I see, it was an addon change.  Someone from the linux port team should take a look at this change.